### DISCUSSION TITLE
How do I import server-handlers in plugins?

### DISCUSSION THREAD

**Discussion by yoshinoriishii** [nitrojs/nitro#1730] [Q&A]
I am trying to develop a plugin by importing server-handlers as follows. 

```
import { handlersMeta } from '#internal/nitro/virtual/server-handlers'
```

Although the compiler gives an error, it seems that handlersMeta can be retrieved when executed. Currently, I am using @ts-expect-error to suppress the error. Is there a better way to handle this?

**Comment by Latex88** [comment]
I was also wondering about this.
Any Updates?
